tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/run/BalxRunFunctionNegativeTestCase.java

```diff
@@ -59,13 +60,33 @@ public void setUp() throws BallerinaTestException, IOException {
                 new LogLeecher[0], tempProjectDir.toString());
         Path generatedBalx = tempProjectDir.resolve("entry.balx");
         balxPath = generatedBalx.toString();
+
```

Comment:
Shall we remove this extra line?

Reply:
this was left on purpose since the above and below blocks are separate setup requirements for two tests, will add a comment instead. Same with the positive test case.